yorkie/src/main/kotlin/dev/yorkie/document/json/JsonTree.kt

```diff
@@ -152,30 +152,29 @@ public class JsonTree internal constructor(
 
         val fromPos = target.findPos(fromIndex)
         val toPos = target.findPos(toIndex)
-        editInternal(fromPos, toPos, contents.toList(), splitLevel)
+        editInternal(fromPos, toPos, splitLevel, *contents)
     }
 
     private fun editInternal(
         fromPos: CrdtTreePos,
         toPos: CrdtTreePos,
-        contents: List<TreeNode>,
         splitLevel: Int = 0,
+        vararg contents: TreeNode,
     ) {
         if (contents.isNotEmpty()) {
-            validateTreeNodes(contents)
-            if (contents.first().type != DEFAULT_TEXT_TYPE) {
-                contents.forEach { validateTreeNodes(listOf(it)) }
-            }
+            validateTreeNodes(*contents)
```

Comment:
@7hong13 I made some fixes here.
before it was validating the same nodes already validated just before.
I think it should validate on the children nodes.